### PR TITLE
Revise Checkout Messaging

### PR DESCRIPTION
**Pull request by tubbo**
Change the messaging in the payment step of checkout to some copy text that Affirm provided for us. This gives a bit more information about Affirm's payment and how it works.

### CODE REVIEW

File: test/system/workarea/storefront/affirm_logged_in_checkout_system_test.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ module Workarea
   module Storefront
     class AffirmLoggedInCheckoutSystemTest < Workarea::SystemTest
       include Storefront::SystemTest
+      include ActionView::Helpers::SanitizeHelper
```
Comment (bencrouse) [inline test/system/workarea/storefront/affirm_logged_in_checkout_system_test.rb]: Why did you need this?

**Review by bencrouse — COMMENTED** [review]
Curious about that sanitize helper.

File: test/system/workarea/storefront/affirm_logged_in_checkout_system_test.rb (hunk 1 above)
Comment (tubbo) [reply test/system/workarea/storefront/affirm_logged_in_checkout_system_test.rb]: ahh sorry this is a relic from my testing and I don't actually need it anymore (I was trying to see if `strip_tags` on the HTML i18n string would match with Capybara's `assert_text` assertion)

**Review by mttdffy — APPROVED** [review]

**Review by bencrouse — APPROVED** [review]
